### ISSUE TITLE
数据的统计

### ISSUE THREAD

**Issue by Book1996** [liu-jc/PyTorch_NGCF#1]
你好, 我最近也在复现这篇论文. 但是我遇到了一个问题, 我所预处理的数据的统计和论文差别很大. 请问, 你有注意到吗

**Comment by ivanchiao** [comment]
Yes,  I did the same with the data prep. And I found the problem too.  
But I don't know how the original text was prepped.

**Comment by liu-jc** [comment]
Hi,
I used the data provided in the original repository. So, I am not sure whether the dataset statistic is the same as that mentioned in the paper. How you pre-process the data? Did you pre-process the data as described in the original paper?
